website/docs/config_guide.md

```diff
@@ -148,6 +148,27 @@ jbrowse add-assembly myfile.fa --load copy --out /var/www/html/jbrowse2
 
 See our CLI docs for the add-assembly for more details here -- [add-assembly](cli#jbrowse-add-assembly-sequence)
 
+Note: assemblies can also be added graphically using the assembly manager. See the quickstart guide for more details. (TODO: link to quickstart)
+
+## Adding tracks and connections
+
+To add a new track or connection, use the menu bar in the app
```

Comment:
if we had one or two screenshots here, specifically for the add track, it would be good. I might remove mention of connections for now actually just to make it focused

Reply:
also this is the config guide, please move to the user_guide.md

Reply:
I can add some screenshots, will work on that

Reply:
Can you move this context to the user_guide.md. I want to make sure the process of adding tracks is seen for unpriviledged end users too